Makefile

```
@@ -10,11 +10,12 @@ TESTS_PATH := tests
 -include .env
 
 ifndef UV_VERSION
-	UV_VERSION := 0.9.11
+	UV_VERSION := 0.9.12
 endif
 
 .PHONY: uv_check venv sync update format lint test docs docs-server docs-format docs-lint release
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)